File: Week_1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompts user for a size
    int size;
    do
    {
        size = get_int("Size: ");
    }
    while (size < 1);

    // builds structure based on size
    for (int i = 0; i < size; i++)
    {
        for (int j = 0; j < size; j++)
        {
            if (size - j - 1 > i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: Week_6/mario.py
from cs50 import get_int

# prompts user for a size
while True:
    size = get_int("Size: ")
    if size <= 8 and size > 0:
        break

# builds structure based on size
for i in range(size):
    n = i+1
    print(" "*(size-n) + "#"*n + "  " + "#"*n)
